boards/arc/nsim/nsim_hs_sram.dts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022, Synopsys, Inc. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "nsim.dtsi"
+#include "nsim-flash-sram-mem.dtsi"
```

Comment:
As we don't use flash in this configuration:

```suggestion
#include "nsim-flat-mem.dtsi"
```

PS: probably we'll rename `nsim-flat-mem.dtsi` to `nsim-sram-mem.dtsi` for less cryptic name :)